File: helium-python/src/systemtest/python/helium_systemtest/environment.py
from build_environment import get_target_path, get_proj_path
from os.path import pardir

def get_test_library():
	return get_target_path('test-dist/heliumlib')

def get_systemtest_resource(*rel_path):
	return get_proj_path(pardir, 'src', 'systemtest', 'resources', *rel_path)

def get_test_dist_dir():
	return get_target_path('test-dist')

File: helium-java/src/systemtest/java/helium_systemtest/Environment.java
package helium_systemtest;

import java.io.File;

import static com.heliumhq.api_impl.application_context.DevelopmentAPIConfig.
		getHeliumFile;

public class Environment {
	public static String getTargetDir() {
		return getHeliumFile("helium-java/target");
	}
	public static String getTestDistDir() {
		return getHeliumFile("helium-java/target/test-dist");
	}
	public static String getTestDistFile(String relPath) {
		return getHeliumFile("helium-java/target/test-dist/" + relPath);
	}
	public static String getSystemtestResource(String relPath) {
		String result =
			getHeliumFile("helium-java/src/systemtest/resources/" + relPath);
		if (new File(result).exists())
			return result;
		return getHeliumFile("src/systemtest/resources/" + relPath);
	}
}
